vader_planner/scripts/run_planner_tester.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# Planner Tester Automation Script
+# This script runs inside the Docker container to automate RRT* parameter testing
+# Usage: ./run_planner_tester.sh [num_trials]
+
+set -e
+
+# Default number of trials
+NUM_TRIALS=${1:-50}
+
+echo "=========================================="
+echo "Planner Tester - RRT* Parameter Testing"
+echo "=========================================="
+echo "Number of trials: $NUM_TRIALS"
+echo "Current directory: $(pwd)"
+echo "=========================================="
+
+# Navigate to workspace
+cd /root/catkin_ws
```

Comment:
```suggestion
cd /root/docker_ws
```
at least how vader_docker is setup, that's the folder